examples/skip_flags/README.md

```diff
@@ -57,4 +57,32 @@ To skip a particular labeled feature , do the following
 
 ```shell
 ./skipflags.test --skip-labels "env=prod"
-```
\ No newline at end of file
+```
+
+### Skip tests using built in --skip flag in go test 
```

Comment:
Is  the new flag `-skip` or `--skip` or it does not matter ?

Reply:
i think needs to be just `-skip` 

`The go test command now accepts -skip <pattern> to skip tests, subtests, or examples matching <pattern>.`
https://tip.golang.org/doc/go1.20

Reply:
+1 for `-skip`

Reply:
fixed using `-skip`